R-package/R/lgb.Booster.R

```diff
@@ -873,7 +886,10 @@ lgb.save <- function(booster, filename, num_iteration = NULL) {
   }
 
   # Store booster
-  invisible(booster$save_model(filename, num_iteration))
+  invisible(booster$save_model(
+    filename = filename
+    , num_iterations = num_iteration
```

Comment:
```suggestion
    , num_iteration = num_iteration
```

this parameter is called `num_iteration`